docs/updating/updating_to_3.3.md

```diff
@@ -0,0 +1,138 @@
+# Updating [[= product_name =]] to v3.3
+
+[[= product_name =]] v3.3 uses [Symfony Flex](https://symfony.com/doc/current/quick_tour/flex_recipes.html).
+When updating from v3.2 to v3.3, you need to follow a special update procedure.
+
+If you are updating from an earlier version, start with updating your installation to v3.2.
+
+First, create an upgrade branch and commit your work.
```

Comment:
```suggestion
First, create an upgrade branch in a VCS and commit your work.
```
BTW, I do not think we ever explicitly state that our VCS of choice is git/GitHub. I know it is obvious, but still, a one sentence intro would be elegant.

Reply:
In a way we mention it in the installation guide, but let's spell it out here.